src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -1612,8 +1613,7 @@ public void hideTitle()
 	@Override
 	public void updateInventory()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		// Pretend packet gets sent.
```

Comment:
I mean... sure, I guess

Reply:
Not sure what else it'd do as all Bukkit does is send a `ClientboundContainerSetContentPacket` (https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java#869)

Reply:
Yeah, it's not like we can really do much in there. It just always feels weird to leave methods like that.
But like I said "sure, I guess". It's not like there is a good alternative :D